recipe_batches/recipe_batches.py

```diff
@@ -6,19 +6,30 @@ def recipe_batches(recipe, ingredients):
   # compare the keys in ingredients to the keys in recipe
   # if a key is missing print 0
   # if all keys are present check how many times the ingredients can be divided into the recipe
-  pass
+  count = 0
+  list_ = []
+  if len(recipe) != len(ingredients):
+    return 0
+  for itemRecipe, numRecipe in recipe.items():
+    for itemIngredients, numIngredients in ingredients.items():
+      if itemRecipe == itemIngredients:
+        if numIngredients / numRecipe < 1:
+          return 0
```

Comment:
great job, this will break you out as soon as you know that you can't create any batches